jenkins-scripts/lib/project-default-devel-homebrew-amd64.bash

```diff
@@ -136,26 +136,21 @@ if brew ruby -e "exit ! '${PROJECT_FORMULA}'.f.recursive_dependencies.map(&:name
   CMAKE_ARGS='-DQWT_WIN_INCLUDE_DIR=/usr/local/opt/qwt-qt5/lib/qwt.framework/Headers -DQWT_WIN_LIBRARY_DIR=/usr/local/opt/qwt-qt5/lib'
 fi
 # Workaround for cmake@3.21.4: set PATH
-if brew ruby -e "exit ! '${PROJECT_FORMULA}'.f.recursive_dependencies.map(&:name).keep_if { |d| d == 'cmake@3.21.4' }.empty?"; then
+if brew ruby -e "exit ! '${PROJECT_FORMULA}'.f.recursive_dependencies.map(&:name).keep_if { |d| d == 'osrf/simulation/cmake@3.21.4' }.empty?"; then
   export PATH=/usr/local/opt/cmake@3.21.4/bin:${PATH}
 fi
 # Workaround for ffmpeg 4: set CMAKE_PREFIX_PATH and PKG_CONFIG_PATH if we are using ffmpeg@4
```

Comment:
```suggestion
# Workaround for ffmpeg 4: set PKG_CONFIG_PATH if we are using ffmpeg@4
```
Or does it need the CMAKE_PREFIX_PATH too?

Reply:
good catch; I applied your suggestion